src/qt/tradehistorydialog.cpp

```diff
@@ -103,7 +94,7 @@ TradeHistoryDialog::TradeHistoryDialog(QWidget *parent) :
     ui->tradeHistoryTable->setContextMenuPolicy(Qt::CustomContextMenu);
     ui->tradeHistoryTable->setHorizontalScrollBarPolicy(Qt::ScrollBarAlwaysOff);
     ui->tradeHistoryTable->setVerticalScrollBarPolicy(Qt::ScrollBarAlwaysOn);
-    Update(); // make sure we're populated before attempting to resize to contents
+    update(); // make sure we're populated before attempting to resize to contents
```

Comment:
Haha. Yeah.. I noticed it too late, and we have several upper case method names. Renaming this one was also just done for consistency, because `contextualMenu()`, `showDetails()` and `copyTxID()` were also lower case.

Maybe we should define some common practise here, like: all class methods lower case, all independent functions upper case?